packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -489,25 +490,26 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
             let networkTime = -1; // responseEnd - startTime
             const spReqDuration = response.headers.get("sprequestduration");
 
-            const resources1 = performance.getEntriesByType("resource");
+            // getEntriesByType is only available in browser performance object
+            const resources1 = performance.getEntriesByType?.("resource") ?? [];
```

Comment:
fine to unblock but @jatgarg mentioned in the original review that this would run in a node environment, so this would probably still want a node implementation out of common-utils

Reply:
Makes sense, but I couldn't find a node equivalent for this.. maybe we make a separate issue for it?